theme/views/@layout.latte

```diff
@@ -9,6 +9,8 @@
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
 	{block styles}
 		<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp}index.css">
+		<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp}print.css" media="print">
+		<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp}oldie.css">
```

Comment:
Nechci načítat styly pro IE všude. Přidává mi to request a payload navíc.

Reply:
Používá oldie.css někdo? Já třeba všechny styly píšu tak, aby stejný soubor fungoval pro všechny browsery a extra oldie.css tím pádem není potřeba.